src/main.js

```diff
@@ -126,13 +130,14 @@ viewSaved.addEventListener("click", switchToSaved);
 takeMeBack.addEventListener("click", switchToMain);
 backToMain.addEventListener("click", switchToMain);
 showPoster.addEventListener("click", makeUserPoster);
+savePoster.addEventListener("click", saveMainPoster);
 // functions and event handlers go here 👇
 function newPoster() {
-  var poster = new Poster(images[getRandomIndex(images)], titles[getRandomIndex(titles)], quotes[getRandomIndex(quotes)]);
-  mainImg.src = poster.imageURL;
+  currentPoster = new Poster(images[getRandomIndex(images)], titles[getRandomIndex(titles)], quotes[getRandomIndex(quotes)]);
+  mainImg.src = currentPoster.imageURL;
   mainImg.alt = "Random image not found";
-  mainTitle.innerText = poster.title;
-  mainQuote.innerText = poster.quote;
+  mainTitle.innerText = currentPoster.title;
+  mainQuote.innerText = currentPoster.quote;
```

Comment:
Glad to see you refactor the code and use the variable currentPoster instead of declaring new ones for your instances.